cmd/policy.go

```diff
@@ -77,6 +77,10 @@ func (sys *PolicySys) Remove(bucketName string) {
 
 // IsAllowed - checks given policy args is allowed to continue the Rest API.
 func (sys *PolicySys) IsAllowed(args policy.Args) bool {
+	if sys == nil {
```

Comment:
If we need this nil check, won't it be needed for the other methods on PolicySys as well?

Reply:
No, all other methods are never called when Policy is nil this is only when IsAllowed() is called.

Reply:
@harshavardhana Why `IsAllowed()` is called with uninitialized PolicySys?  It seems there is a bug outside of PolicySys.  Its not a good idea to do `nil` check here.

Reply:
The server is already started with object layer initialized @balamurugana there is no bug here, it is simply that PolicySys is empty at some point in time.

Reply:
PolicySys depends on Object Layer.  Uninitialized PolicySys is also uninitialized minio server i.e. the server is still loading.  Wondering why `PolicySys.IsAllowed()` is called.  Allowing requests at this phase is a bug and security problem.

Reply:
Well that is how the server was up until now.  It needs to be fix differently.. which I will send tonight.

Reply:
Also this PR makes sure that only Owner is allowed.

Reply:
Object layer initialization is not at a problem here.  If you mean object layer initialization as HTTP handlers initialization is problem.  No request other than internal RPC is allowed.

Even a policy may prevent owner to do certain operations and the requests may fall into one of such operation (simple example is no upload is allowed without encryption key).

As this PR opens up a security problem, it would be reverted and fixed properly.